.github/workflows/test-getting-started.yml

```diff
@@ -0,0 +1,39 @@
+name: Test the getting start steps
+
+# run CI on pushes to main, and on all PRs (even the ones that target other
+# branches)
+
+on:
+ push:
+   branches: [$default-branch]
+ pull_request:
```

Comment:
should we also add branches here? this will check if the pull request is made to the master/main branch and prevent the workflow from running on other branches

Reply:
```suggestion
 pull_request:
     branches: [main]
```

Reply:
Fixed.